dot_net/CSharpDotNetSamples/CaptureJson.aspx.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Web;
+using System.Web.UI;
+
+namespace AspNetClientEncryptionExample

```

Comment:
I don't know a lot about C# namespaces but it looks like you are using the AspNetClientEncryptionExample namespace in a lot of places like this that aren't really related to ClientEncryption.  If the same namespace needs to be used everywhere for things to work you might want to consider changing the namespace to something more generic to avoid confusion about it.